sycl/test/matrix/matrix-bfloat16-test.cpp

```diff
@@ -0,0 +1,191 @@
+// RUN: %clangxx -fsycl -O2 %s -o %t.out
+#include <CL/sycl.hpp>
+#if (SYCL_EXT_ONEAPI_MATRIX == 2)
+#include <iostream>
+
+using namespace sycl::ext::oneapi::experimental::matrix;
+using bfloat16 = sycl::ext::oneapi::experimental::bfloat16;
+
+static constexpr size_t TILE_SZ = 16;
```

Comment:
Since your macro was not using any type at the first place, you could just use `auto` instead of `size_t`.
Because with the macro the numbers we interpreted as 32-bit `int` while here you are forcing 64-bit, which might be somewhat less efficient in some case (but not in this test I guess).